push-agents/apns-agent/src/main/java/org/aerogear/push/apns/ios/SimpleApnsClientCache.java

```diff
@@ -0,0 +1,153 @@
+/**
+ * JBoss, Home of Professional Open Source
+ * Copyright Red Hat, Inc., and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * 	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.aerogear.push.apns.ios;
+
+import com.turo.pushy.apns.ApnsClient;
+import io.netty.util.concurrent.Future;
+import io.netty.util.concurrent.GenericFutureListener;
+import net.jodah.expiringmap.ExpirationListener;
+import net.jodah.expiringmap.ExpirationPolicy;
+import net.jodah.expiringmap.ExpiringMap;
+import org.jboss.aerogear.unifiedpush.api.iOSVariant;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.PreDestroy;
+import java.util.Map;
+import java.util.concurrent.ConcurrentMap;
+import java.util.concurrent.TimeUnit;
+
+public final class SimpleApnsClientCache {
+
+    private static final SimpleApnsClientCache INSTANCE = new SimpleApnsClientCache();
+
+    private SimpleApnsClientCache() {
+        // no-op
+    }
+
+    public static SimpleApnsClientCache getInstance(){
+        return INSTANCE;
+    }
+
+    private final Logger logger = LoggerFactory.getLogger(SimpleApnsClientCache.class);
```

Comment:
I usually prefer to declare member variable before the constructors.
Moreover, this should be static